api/routes/projects.js

```diff
@@ -122,4 +122,73 @@ router.get('/', async (req, res, next) => {
   });
 });
 
+router.get('/single', async (req, res, next) => {
+
+  const singleRes = await axios.get(
+    `https://sta2020.atlassian.net/rest/api/3/issue/${req.query.res}`,
+    {
+      headers: {
+        Authorization: `Basic ${Buffer.from(
+          // below use email address you used for jira and generate token from jira
+          `${email}:${api_key}`,
+        ).toString('base64')}`,
+        Accept: 'application/json',
+      },
+    },
+  )
+  // .then((resData) => {
+
+  //   project = {
+  //     res_id: resData.data['id'],
+  //     it_related_field_id: resData.data['fields'].customfield_10109,
+  //     jobRole: resData.data['fields'].customfield_10113,
+  //     projectType: resData.data['fields'].customfield_10112,
+  //     suitableForBuddy: resData.data['fields'].customfield_10108 ? resData.data['fields'].customfield_10108.value : 'none',
+  //     candidateTime: resData.data['fields'].customfield_10062 ? resData.data['fields'].customfield_10062 : 'none',
+  //     candidateCoreSkills: resData.data['fields'].customfield_10061 ? resData.data['fields'].customfield_10061 : 'none',
+  //   }
+ 
+  // }).then(() => {
+    const singleIt =  axios.get(
+      `https://sta2020.atlassian.net/rest/api/3/issue/${req.query.it}`,
+      {
+        headers: {
+          Authorization: `Basic ${Buffer.from(
+            // below use email address you used for jira and generate token from jira
+            `${email}:${api_key}`,
+          ).toString('base64')}`,
+          Accept: 'application/json',
+        },
+      },
+    )
+    // console.log(singleIt);
+  // }).then((itData) => {
+  //   // console.log(itData);
+  //   // project.it_key = itData.data['key'];
+  //   // project.projectName = itData.data['fields'].summary;
+  //   // project.charityName = itData.data['fields'].customfield_10027;
+  //   // project.charityVideo = itData.data['fields'].customfield_10159 ? itData.data['fields'].customfield_10159 : 'none';
+  // })
+  //   res.status(200).json(project)
+
+ const [resResults, itResults] = await Promise.all([singleRes, singleIt])
+  const project = {
+        res_id: resResults.data.id,
+        it_related_field_id: resResults.data.fields.customfield_10109,
+        jobRole: resResults.data.fields.customfield_10113,
+        projectType: resResults.data.fields.customfield_10112,
+        suitableForBuddy: resResults.data.fields.customfield_10108 ? resResults.data.fields.customfield_10108.value : 'none',
+        candidateTime: resResults.data.fields.customfield_10062 ? resResults.data.fields.customfield_10062 : 'none',
```

Comment:
If you wanted to, I think you could simplify this and some of the other lines below that follow a similar format, by using a [nullish coalesce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) e.g. `candidateTime: resResults.data.fields.customfield_10062 ?? : 'none',`

Reply:
Sorry, just realised there was a typo in my comment -- that could be what was causing you problems!  I think `candidateTime: resResults.data.fields.customfield_10062 ?? 'none',` should work? (I'd left in a stray `:` after  the`??` -- my bad 🤦‍♂️ )